ios/hdfs/pom.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Sure, I planned to do several commits in the same PR, but I can do several PRs, no problem at all.